core/src/main/resources/filodb-defaults.conf

```diff
@@ -25,9 +25,12 @@ filodb {
     columns = ["_metric_:string", "tags:map"]
 
     # Predefined keys allow space to be saved for over the wire tags with the given keys
-    # WARN: It is suggested to only ADD to predefined keys.
-    #       Changing/Renaming existing keys will cause incorrect tags to be reported and probably require a clean wipe.
-    predefined-keys = ["_ws_", "_ns_", "app", "__name__", "instance", "dc", "le", "job", "exporter", "_step_"]
+    # WARN: 1. It is suggested to only ADD to predefined keys that are not used already,
+    #          which means only new tags with underscore prefixed and suffixed.
+    #       2. Changing/Renaming existing keys will cause incorrect tags to be reported and probably require a clean wipe.
+    #       3. Even when adding new keys, be careful. If that tag is already being used, there is no
+    #          equality of partKeys between old and new time series.
+    predefined-keys = ["_ws_", "_ns_", "app", "__name__", "instance", "dc", "le", "job", "exporter", "_pi_"]
```

Comment:
Comments for what these special keys mean? I cannot find any references to `_pi_`

Reply:
`_pi_` stands for Publish interval. It is intentionally a new unused tag name so the new time series can be differentiated from the ones that already have `_step_` tag.

Reply:
Will add docs.